contrib/cf-remote/cf_remote/ssh.py

```diff
@@ -20,11 +22,25 @@ def connect(host, users=None):
     for user in users:
         try:
             c = fabric.Connection(host=host, user=user)
+
+            try:
+                c.open()
+            except (SSHException, ConnectionResetError, EOFError):
```

Comment:
So the two new exceptions here are in the default Python's global scope?

Reply:
Yes, both are from https://docs.python.org/3/library/exceptions.html

Reply:
We could consider dropping this commit, I don't think it's needed after I made the other changes with reusing the connection.

Reply:
> We could consider dropping this commit, I don't think it's needed after I made the other changes with reusing the connection.

Cool, the fewer "random" exceptions we catch, the better.